src/components/Navbar/Navbar.jsx

```diff
@@ -1,11 +1,19 @@
 import React from "react";
 import DesktopNav from "./ApplicationHeader";
 import MobileNav from "./MobileApplicationHeader";
+import { Hidden } from "@material-ui/core";
 
 const Navbar = () => {
-  const mql = window.matchMedia("(max-width: 1200px)");
-  let mobileView = mql.matches;
-  return <>{mobileView ? <MobileNav /> : <DesktopNav />}</>;
+  return (
+    <>
+      <Hidden smDown>
+        <DesktopNav />
+      </Hidden>
+      <Hidden mdUp>
+        <MobileNav />
+      </Hidden>
+    </>
```

Comment:
maybe ```mdDown``` and ```lgUp``` is better on this. Since then i don't think we will loose the logo at the top on smaller screens :)